src/components/views/auth/InteractiveAuthEntryComponents.tsx

```diff
@@ -606,6 +606,93 @@ export class MsisdnAuthEntry extends React.Component<IMsisdnAuthEntryProps, IMsi
     }
 }
 
+interface IRegistrationTokenAuthEntryState {
+    registrationToken: string;
+}
+
+export class RegistrationTokenAuthEntry extends React.Component<IAuthEntryProps, IRegistrationTokenAuthEntryState> {
+    public static readonly LOGIN_TYPE = AuthType.RegistrationToken;
+
+    constructor(props) {
```

Comment:
sorry, code style changed between last review and now:
```suggestion
    public constructor(props: IAuthEntryProps) {
```